app/controllers/jobs_controller.rb

```diff
@@ -5,7 +5,8 @@ def index
     if current_user && params[:my_jobs]
       @jobs = Job.find(current_user.likedjobs.pluck(:job_id))
     else
-      @jobs = Job.order(posting_date: :desc).includes(:company)
+      @jobs = Job.where.not(id: current_user.jobshidden.pluck(:id))
+        .order(posting_date: :desc).includes(:company)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.